docs/api-ref.md

```diff
@@ -50,7 +50,7 @@ Name | Description
 :--- | :---   
 `username` | The name of the user whose credentials will be used to sign in.   
 `password` | The password of the user.     
-`site_id` | This corresponds to the `contentUrl` attribute in the Tableau REST API. The `site_id` is the portion of the URL that follows the `/site/` in the URL. For example, "MarketingTeam" is the `site_id` in the following URL *MyServer*/#/site/**MarketingTeam**/projects. To specify the default site on Tableau Server, you can use an empty string `''`  (single quotes, no space).  For Tableau Online, you must provide a value for the `site_id`.  
+`site_id` | This corresponds to the `contentUrl` attribute in the Tableau REST API. It is the portion of the URL that follows the `/site/` in the URL. For example, "MarketingTeam" is the `site_id` in the following URL: `https://MyServer/#/site/MarketingTeam/projects`. To specify the default site on Tableau Server, you can either omit the site_id or use an empty string `''`  (single quotes, no space).  For Tableau Online, you must provide a value for the `site_id`.  
```

Comment:
can we change this from 'site_id' to site_url_id or something? Then all the other uses of id should be luids.